blog/20131125-support-linux-voice.md

```diff
@@ -8,17 +8,17 @@
 .. author: Martin Wimpress
 -->
 
-The MATE team encourages you to head over to [Linux Voice on indiegogo](http://www.indiegogo.com/projects/linux-voice)
-and find out why you, as an Open Source user and enthusiast, should back [Linux Voice](http://www.linuxvoice.com/).
+The MATE team encourages you to head over to [Linux Voice on indiegogo](https://www.indiegogo.com/projects/linux-voice)
+and find out why you, as an Open Source user and enthusiast, should back [Linux Voice](https://www.linuxvoice.com/).
```

Comment:
same here

Reply:
Same here, i got a secure warning from firefox when using https://www.linuxvoice.com/
, removing the link.

Reply:
Hmm, i think this is from Martin and his blog.
Do you agree with leaving the links `https://www.linuxvoice.com/` here.
People will get only a secure warning by firefox, and they can decide for themself to follow the link or not.

Reply:
ah yes, the web works, https://www.linuxvoice.com/ is ok for me